src/public/styles.css

```diff
@@ -154,3 +168,38 @@ nav > ul {
   font-weight: lighter;
   margin-top: 30px;
 }
+
+.password-ul {
+  /* border: 2px solid red; */
+  margin-left: -50px;
+  margin-top: 0px;
+}
+
+ul li {
+  list-style-type: none;
+}
+
+i {
+  margin: 0.5rem;
+}
+
+.submit-btn {
+  align-self: center;
+}
+
+/* login.html */
+
+.flex-center-login-form {
```

Comment:
You don't need to change this but another approach might be to separate this into two classes. You _could_ _perhaps_ have a utility class called flex-centre. and a class called login-form. Then add the reusable flex centre stuff to that class and anything only applying to the login form to the other class. 

Otherwise, if you were to leave as is i'd personally rename the class, I don't think you need to have any of the css properties in the class name unless it's a utility class. But that's just my preference and you may well disagree :).